JSDemos/Demos/TreeList/FormEditing/description.md

```diff
@@ -1 +1,19 @@
-This example demonstrates editing in _&laquo;form&raquo;_ mode. To&nbsp;begin to&nbsp;edit a&nbsp;row, click its _&laquo;Edit&raquo;_ link. On&nbsp;switching into the editing state, this row will become a&nbsp;form with editable fields.
\ No newline at end of file
+TreeList can use the [Form](/Documentation/ApiReference/UI_Components/dxForm/) component to arrange cell editors when users modify a row. Users can edit values in [visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#visible) and hidden columns.
```

Comment:
From what I understand, the editors on the Form allow users to modify values both in visible and hidden columns. However, the second sentence here does not convey this idea clearly enough. Right now to me it sounds like users always can edit all kinds of columns. Please take another look at this paragraph. You most likely can come up with something better.